src/globals/scss/vendor/@carbon/elements/scss/layout/_convert.scss

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright IBM Corp. 2018, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+// Default, Use with carbon--em() and carbon--rem() functions
+$carbon--base-font-size: 16px !default;
+
+/// Convert a given px unit to a rem unit
+/// @param {px} $px
+/// @return {rem}
+@function carbon--rem($px) {
```

Comment:
It hadn't even occurred to me to add carbon to our function names. Will we be having to update our code where we're currently using `rem()`?

Reply:
I think we can just keep using `rem` in the codebase and just alias out to the layout function, at least that's the hope!